DynamicFeatureNavigation/app/build.gradle.kts

```diff
@@ -53,4 +57,34 @@ dependencies {
     api("androidx.navigation:navigation-dynamic-features-fragment:2.3.0-alpha01")
     api("androidx.appcompat:appcompat:1.1.0")
     api("androidx.constraintlayout:constraintlayout:1.1.3")
+    implementation("com.google.android.play:core:1.6.5")
```

Comment:
`androidx.navigation:navigation-dynamic-features-fragment` exposes 1.6.4 of the library.
Filed an issue to [update it in androidx](https://issuetracker.google.com/149556401)